src/lib/storage-utils.js

```diff
@@ -71,11 +71,34 @@ function cleanUpReadme(version: Version): Version {
   return version;
 }
 
+function getLatestReadme(pkg: Package): string {
+  const versions = pkg['versions'] || {};
+  const distTags = pkg['dist-tags'] || {};
+  const latestVersion = distTags['latest'] ? versions[distTags['latest']] || {} : {};
+  let readme = latestVersion.readme || pkg.readme || '';
+  // In case of empty readme - trying to get ANY readme in the following order: 'next','beta','alpha','test','dev','canary'
+  const readmeDistTagsPriority = [
```

Comment:
If you catch here `readme` you do not need to iterate.
`let readme = latestVersion.readme || pkg.readme || '';`

Please return readme whether is different than an empty string. I'd add `|| ''` only as las resource.

Try to avoid iterations if are not need it.